content/index.fr.md

```diff
@@ -0,0 +1,232 @@
+
+[[Modèle de rapport d'évaluation de l'accessibilité Web]](/test-evaluate/report-template/)
+:   Propose des informations à inclure dans un rapport.
+
+[Outil de reporting ATAG](https://www.w3.org/WAI/atag/report-tool/) 
+:   Vous aide à générer un rapport pour évaluer dans quelle mesure un outil répond aux critères des Règles d'accessibilité pour les outils d'édition([ATAG]) (https://www.w3.org/WAI/standards-guidelines/atag/). Les outils d'édition sont des logiciels et des services utilisés pour créer du contenu Web, tels que les systèmes de gestion de contenu (CMS en anglais) et les éditeurs HTML WYSIWYG (what-you-see-is-what-you-get, "ce que vous voyez est ce que vous obtenez").
+
+Communique clairement les résultats de l'évaluation, y compris les objectifs de l'évaluation.
+
+## Personnes
+
+{% include video-card.html
+    video-url="//media.w3.org/wai/evaluation-intros/involving-users.mp4"
+    captions="/content-images/wai-eval-overview/involving-users-cc.vtt|en|default"
+    accessible-version="https://www.w3.org/WAI/test-evaluate/involving-users/#video-intro"
+    accessible-version-label="Vidéo : vue d'ensemble : impliquer des utilisateurs dans l'accessibilité Web"
+    poster="/content-images/wai-eval-overview/video-thumb-involving-users.png"
+%}
+
+Impliquer les personnes et les compétences adéquates rendra votre évaluation de l'accessibilité plus efficace.
+
+[[Impliquer des utilisateurs dans l'évaluation de l'accessibilité Web]](/test-evaluate/involving-users/)
+:   Donne des conseils pour inclure des personnes en situation de handicap ("utilisateurs") dans l'évaluation tout au long du développement du projet.
+
+[Une expertise combinée pour évaluer l'accessibilité Web](/test-evaluate/combined-expertise/) 
+:   Présente différentes aptitudes et considérations pour une vérification collaborative.
+
+## Standards
+
+Des spécificités liées à l'écriture de règles de test de l'accessibilité (ACT Rules Format) et à l'expression des résultats de test (EARL) sont introduites dans la [Vue d'ensemble des standards d'évaluation](/standards-guidelines/evaluation/) 
+
+<hr>
+
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+##  Vue d'ensemble des ressources - Introduction aux alternatives vidéo {#alternatives}
+
+{% include excol.html type="middle" %}
+
+_Cette vidéo est aussi disponible sur un serveur du W3C : [Vidéo: Evaluation de l'accessibilité Web - Vue d'ensemble des ressources (format du fichier : MP4, taille du fichier: 45MB)](http://media.w3.org/wai/evaluation-intros/evaluating-resources-overview.mp4)._
+
+###  Transcription avec audiodescription pour la vidéo d'introduction - Vue d'ensemble des ressources {#transcript}
+
+<table aria-labelledby="transcription">
+  <tbody>
+    <tr>
+      <th>Audio</th>
```

Comment:
Sylvie's ideas for translation of the audio are included in the [VTT file](https://github.com/w3c/wai-eval-overview/blob/master/content-images/wai-eval-overview/evaluating-resources-overview-fr.vtt)

This table and the VTT file translation should match. However, that is low priority. I put an [issue](https://github.com/w3c/wai-eval-overview/issues/40) so we don't forget.

Reply:
Indeed, they don't exactly match for now.